testbed/testbed/utils.go

```diff
@@ -23,3 +23,13 @@ import (
 func GetAvailablePort(t *testing.T) int {
 	return int(testutils.GetAvailablePort(t))
 }
+
+// Structure used for TestTraceNoBackend10kSPS.
+// Defines RAM usage range for defined processor type.
+type ProcessorConfig struct {
+	Name string
+	// map of processor types to their config YAML to use.
+	ProcessorConfig     map[string]string
+	ExpectedMaxRAM      uint32
+	ExpectedMinFinalRAM uint32
+}
```

Comment:
This is used by code in `testbed/tests`. Please move this closer to where it is needed, perhaps close to `ScenarioTestTraceNoBackend10kSPS` which uses it (or in a separate .go file if it needs to be used by many functions from different files in that package). There does not seem to be a need to declare this in the testbed at the moment.